dcpy/lifecycle/builds/clean_artifacts.py

```diff
@@ -55,7 +55,7 @@ def delete_stale_image_tags(active_build_names: list[str]):
         ).json()["results"]
         dev_tags = [tag for tag in tags if tag["name"].startswith("dev-")]
         for tag in dev_tags:
-            if tag not in active_build_names:
+            if tag[4:] not in active_build_names:
```

Comment:
seems safer to use the prefix rather than an index, mayb

```python
if tag.removeprefix("dev-") not in active_build_names:
```

I was gonna say `tag.split("dev-")[1]`, but a case like `dev-dev-my-branch` ruined that